server/src/daos/timeline.js

```diff
@@ -0,0 +1,67 @@
+import Timeline from '../models/timeline';
+
+export const getTimelineById = async (id) => {
+  const response = {
+    data: {},
+    error: null,
+  };
+  try {
+    response.data = await Timeline.findById(id);
+  } catch (err) {
+    response.error = err.msg;
```

Comment:
this will return a valid response even for errors, and frontend will get 200 instead of respective errors

Reply:
Will handle that